packages/ui/src/index.ts

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 //
 
-import { UIComponent } from 'svelte'
+import { UIComponent } from '@anticrm/status'
```

Comment:
I supposed this change was already merged?

Reply:
Yeah, will disappear after rebase